public/index.html

```diff
@@ -13,7 +13,20 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Technigo React App</title>
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+    </style>
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Montserrat&family=Roboto:wght@700&display=swap');
+    </style>
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@500;600&display=swap');
+    </style>
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
+    </style>
+    <link rel="icon" type="image/png" sizes="32x32" href="./favicon-32x32.png">
+    <title>Jessika Lind</title>
```

Comment:
Here I don't exactly understand but it seems that both Roboto and Monserrat is imported more than once? Different weights but they can be imoprted all at once with that which maybe would make tthis look a bit cleaner.